src/xcode/ENA/.swiftlint.yml

```diff
@@ -240,7 +241,7 @@ type_body_length:
   error: 400
   
 function_parameter_count:
-  warning: 6
+  warning: 7
```

Comment:
Personally, I think even 5 parameters is too much for normal code. It may be valid in linear algebra and graphics programming. Here we should consider configuration structs/models to be passed into bigger functions.